Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -288,16 +276,28 @@ final class Analytics {
         track(event)
     }
     
-    func introClick(_ label: Label.Navigation, at page: Property.OnboardingPage?) {
-        guard let page,
-              let pageAsInt = Property.OnboardingPage.allCases.firstIndex(of: page) else {
+    func introDisplaying(page: Property.OnboardingPage?, index: Int) {
```

Comment:
Interesting point 👍 .
I guess the root reason for having the `index` instead of retrieving it from the page itself was due to the fact that the onboarding sections might change depending on the market being redirected to?

Reply:
That is also a good point, in my opinion, even if they might change, we are still mostly interested in where the page is located, which enables us to know, for example, which interactions happen on the first page independent of what it is. Therefore, I believe changing it to the actual index does bring some maintainability benefits while still working as expected at the moment.

Reply:
More explicit 👌 - thanks for the explanation. No strong opinions on that